app/controllers/api/users.php

```diff
@@ -1420,6 +1422,186 @@ function createUser(string $hash, mixed $hashOptions, string $userId, ?string $e
             ->dynamic($target, Response::MODEL_TARGET);
     });
 
+App::post('/v1/users/:userId/sessions')
+    ->desc('Create session')
+    ->groups(['api', 'users'])
+    ->label('event', 'users.[userId].sessions.create')
+    ->label('scope', 'users.write')
+    ->label('audits.event', 'session.create')
+    ->label('audits.resource', 'user/{request.userId}')
+    ->label('usage.metric', 'sessions.{scope}.requests.create')
+    ->label('sdk.auth', [APP_AUTH_TYPE_KEY])
+    ->label('sdk.namespace', 'users')
+    ->label('sdk.method', 'createSession')
+    ->label('sdk.description', '/docs/references/users/create-user-session.md')
```

Comment:
We are missing this markdown file. Also lets rename to just `create-session`, its in `users` folder already.

I think it will be worth doing a link between `create-session` and `create-token`, and explain difference. For example something like `Use this to create session for user without authentication flow. bla bla bla... If you instead want to include authentication flow, please use [link to createToken] instead.`. And something similar for create token, linking here if you want to go without authentication.

Reply:
Added